pkgs/tools/text/textpieces/default.nix

```
@@ -61,6 +61,8 @@ stdenv.mkDerivation (finalAttrs: {
     pythonEnv
   ];
 
+  patches = [ ./change_float_to_nearest_integer.patch ];
```

Please submit a patch upstream and fetch it
```suggestion
  patches = [ (fetchpatch {
    url = "...";
    hash = "...";
  }) ];
```